pkg/plugins/policies/meshtimeout/plugin/v1alpha1/plugin.go

```diff
@@ -83,9 +90,12 @@ func applyToInbounds(fromRules core_rules.FromRules, inboundListeners map[core_r
 		}
 
 		protocol := core_mesh.ParseProtocol(inbound.GetProtocol())
+		conf := getConf(fromRules.Rules[listenerKey], core_rules.MeshSubset())
+		if conf == nil {
```

Comment:
I'm pretty sure if the `conf` is nil we still have to let the ListenerConfigurer to work. For example, the idleTimeout here is set to 0 regardless of `conf`: https://github.com/kumahq/kuma/blob/dd29be83bc55f7620a6f9aec407f0f0dbbb5ccd7/pkg/plugins/policies/meshtimeout/plugin/xds/configurer.go#L47

Reply:
changed to using deafult conf

Reply:
why haven't we caught that in unit tests? are we missing some cases when conf is nil?

Reply:
we have unit test for it, but it was working without this. I think that this is because by default on resource it is 0, and we are setting it to 0 if there is no conf. Looking at the commit history it was needed to override old Timeout policy. And don't think we should add unit test for overriding old policies as they should not be mixed anyway